environment.yml

```diff
@@ -1,6 +1,6 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-hic-1.4.0
+name: nf-core-hic-2.0.0
 channels:
   - conda-forge
```

Comment:
Do you still need that file?

Reply:
You should be able to use the individual conda envs from the modules instead. No?

Reply:
never tried like this. If you want I can remove it ?

Reply:
Try running `nextflow run . -profile test,conda` in your hic folder, if it works well for you, we can remove this file.
I'd just rather avoid having unnecessary files

Reply:
@maxulysse, I tried to run the pipeline with `-profile test,conda` ... and I was not able to get it running until the end. I do not think this is an issue with the pipeline itself but more with `conda` in general. I frequently have some memory conda issue ... especially when it tries to run several env in the same time.
This, I think that this all-in-one environment could still be useful for some users ... but if this is no longer in the nf-core good practices, I can remove it

Reply:
Btw, I have a warning I do not manage to fix, would you have any idea ;

```
WARN: A process with name 'BOWTIE2_ALIGN_TRIMMED' is defined more than once in module script: /home/nservant/GIT/nf-core-hic/./workflows/../subworkflows/local/./hicpro_mapping.nf -- Make sure to not define the same function as process
```

Reply:
> @maxulysse, I tried to run the pipeline with `-profile test,conda` ... and I was not able to get it running until the end. I do not think this is an issue with the pipeline itself but more with `conda` in general. I frequently have some memory conda issue ... especially when it tries to run several env in the same time. This, I think that this all-in-one environment could still be useful for some users ... but if this is no longer in the nf-core good practices, I can remove it

I have no issues with keeping the file. You know why you have it, and it's useful for you.

Reply:
> Btw, I have a warning I do not manage to fix, would you have any idea ;
> 
> ```
> WARN: A process with name 'BOWTIE2_ALIGN_TRIMMED' is defined more than once in module script: /home/nservant/GIT/nf-core-hic/./workflows/../subworkflows/local/./hicpro_mapping.nf -- Make sure to not define the same function as process
> ```

I had a look at the subworkflow, but couldn't find anything to explain that.